frontend/scss/pages/_home_page.scss

```diff
@@ -199,6 +199,48 @@
   margin-top: 50px;
 }
 
+.products-component-container {
+  margin: 0 auto;
+}
+
+.products-component {
+  margin: 20px 5px 20px 5px;
+  padding: 5px 5px 5px 5px;
+
+  display: inline-block;
+
+  //background-color: #325774;
+  //below is site color, i prefer the above though
+  background-color: #363f46;
+
+  border: 1px solid white;
+}
+
+.products-component-name {
+  margin-bottom: 2px;
+
+  font-size: 16px;
+  font-weight: bold;
+  color: white;
+
+  clear: both;
+}
+
+.products-component-brand {
+  margin-top: 2px;
```

Comment:
you can define a shared css selector between brand and name, and extend it in both of these:

```
%product-summary-text {
  margin-bottom: 2px;
  font-size: 16px;
  font-weight: bold;
  color: white;
}

.product-summary__name, .product-summary__brand {
  @extend %product-summary-text;
}
```

the % in scss makes it so the css is not actually defined in the % selector, only when it's extended. This keeps the overall css shorter